xcm/xcm-executor/src/config.rs

```diff
@@ -94,4 +95,9 @@ pub trait Config {
 	/// The origin locations and specific universal junctions to which they are allowed to elevate
 	/// themselves.
 	type UniversalAliases: Contains<(MultiLocation, Junction)>;
+
+	/// How calls are dispatched from XCM.
+	/// Allows to modify the origin and/or the call based on the other.
+	/// To simply do `call.dispatch(origin)`, use `JustDispatch`.
+	type CallDispatcher: CallDispatcher<Self::Call>;
```

Comment:
```suggestion
	/// The call dispatcher used by XCM.
	/// 
	/// XCM will use this to dispatch any call. When no special call dispatcher is required, this can be set to the same type as `Self::Call`.
	type CallDispatcher: CallDispatcher<Self::Call>;
```